### PR TITLE
Add ofcir service

### PR DESCRIPTION
**Pull request by derekhiggins**
Add the deployment and role required to run ofcir.

### CODE REVIEW

File: clusters/build-clusters/build05/ofcir/ofcir.yaml [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+      terminationGracePeriodSeconds: 10
+---
+kind: Group
+apiVersion: user.openshift.io/v1
+metadata:
+  name: ofcir-admin
+users:
+- afasano
+- dhiggins
+- tdomnesc
```
Comment (bradmwilliams) [inline clusters/build-clusters/build05/ofcir/ofcir.yaml:549]: The preferred way to do this is via Rover Groups: https://docs.ci.openshift.org/docs/how-tos/rbac/#rover-groups

Comment (derekhiggins) [reply clusters/build-clusters/build05/ofcir/ofcir.yaml]: thanks, group created.

File: clusters/build-clusters/build05/ofcir/ofcir.yaml [hunk 2]
```diff
@@ -0,0 +1,569 @@
...
+  name: ofcir-service
+  namespace: ofcir-system
+spec:
+  ports:
+  - name: http
+    port: 8087
+    protocol: TCP
+  selector:
+    control-plane: controller-manager
+  type: NodePort
```
Comment (andfasano) [inline clusters/build-clusters/build05/ofcir/ofcir.yaml:459]: I think this should work (ie visible by normal Prow jobs running on build05 cluster) also by setting a `ClusterIP` just to limit internally the visibility

Comment (derekhiggins) [reply clusters/build-clusters/build05/ofcir/ofcir.yaml]: done

**Review by jmguzik — APPROVED** [review]

**Pull request merged**